_commands/basics/printf.md

```diff
@@ -23,27 +23,28 @@ $ printf "%d\n " 0xF
 $ printf "%f\n" 5
 ~~~
 
-##### Break it down
+#### Break it down
 
-*This is a simple use of 'printf'. It prints out the number 5 as a float on a new line. The default is to print with 6 decimals.
+* Prints out the number 5 as a float on a new line. The default is to print with 6 decimals.
 
 ~~~ bash
 $ printf "\t\"Quotations\"\t\n"
 	"Quotations"	
 $
 ~~~
 
-##### Break it down
+#### Break it down
 
-*This command takes advantage of printf's ability to use whitespace by adding tabs into the output using \t to add tabs and a newline at the end
-*This command also demonstrates how to add characters to strings that would normally give a command to printf. Quotations usually specify what is meant to be fed into printf, but using \" allows an actual double quote to be inserted into the string.
+* This command takes advantage of the ability for 'printf' to use whitespace by adding tabs into the output using \t to add tabs and a newline at the end
+
+* This command also demonstrates how to add characters to strings that would normally give a command to 'printf'. Quotations usually specify what is meant to be fed into 'printf', but using \\" allows an actual double quote to be inserted into the string.
 
 ~~~ bash
 $ printf "put a number here: %d and a different one here: %d\n" 21 45
 put a number here: 21 and a different one here: 45
 ~~~
 
-##### Break it down
+#### Break it down
 
 *This command takes advantage of being able to insert decimal numbers into strings using the %d to refer to decimals
```

Comment:
One more space here between the `*` and the `This command...`, otherwise looking good!